components/gitpod-db/src/typeorm/migration/1667919924081-CostCenterBillingCycleStart.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const D_B_COST_CENTER = "d_b_cost_center";
+const COL_BILLING_CYCLE_START = "billingCycleStart";
+
+export class CostCenterBillingCycleStart1667919924081 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, D_B_COST_CENTER, COL_BILLING_CYCLE_START))) {
+            await queryRunner.query(
+                `ALTER TABLE ${D_B_COST_CENTER} ADD COLUMN ${COL_BILLING_CYCLE_START} varchar(30) NOT NULL, ALGORITHM=INPLACE, LOCK=NONE `,
```

Comment:
I think we should update the cost centers as well, otherwise this is not going to fix until everyone has renewed. Using `creationDate` would do it > 90%.

Reply:
To get the other 10insh percent we could check existence of credit usage and use the effectiveDate.

Reply:
Thanks for the suggestion! I initially thought it might be okay to just wait one month for all existing free tier CostCenters to fix themselves, but I agree there is value in fixing them sooner.

Follow-up issue: https://github.com/gitpod-io/ops/issues/6712